R/exceptions.R

```diff
@@ -33,4 +33,6 @@
     warnings
 }
 
-
+`str.Rcpp_stack_trace` <- function(x, ...) {
```

Comment:
Almost not worth a commit; we can clean up another time.  We still have inconsistent white-space and whatnot.

Oh, and there he did :)   Thanks.